src/__experimental__/components/radio-button/radio-button.stories.js

```diff
@@ -27,11 +27,45 @@ function makeStory(name, themeSelector, component) {
   return [name, component, metadata];
 }
 
-const radioComponent = () => {
-  const knobs = defaultKnobs();
+const groupedKnobs = (type, themeName) => {
+  return {
+    key: type,
+    label: text(`${type} label`, `Example ${type} radio button`, type),
+    labelHelp: text(`${type} labelHelp`, 'This text provides more information for the label.', type),
+    value: text(`${type} value`, type, type),
+    disabled: boolean(`${type} disabled`, false, type),
+    reverse: boolean(`${type} reverse`, false, type),
+    size: themeName !== 'classic' ? select(`${type} size`, OptionsHelper.sizesBinary, 'small', type) : undefined,
```

Comment:
I recommend inverting the condition:

```suggestion
    size: themeName === 'classic' ? undefined : select(`${type} size`, OptionsHelper.sizesBinary, 'small', type),
```

IMO we should enable ESLint's [`no-negated-condition`](https://eslint.org/docs/rules/no-negated-condition) rule:

> Negated conditions are more difficult to understand. Code can be made more readable by inverting the condition instead.